.github/workflows/multi-arch-test-build.yml

```diff
@@ -0,0 +1,57 @@
+name: Test Build
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: ${{ matrix.arch }} build
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        arch:
+          - x86_64
+          - mips_24kc
+          - arm_cortex-a9_neon
+          - arm_cortex-a9_vfpv3-d16
+          - arc_arc700
+          - aarch64_generic
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 20
+      - run: git fetch --depth=20
+
+      - name: Determine changed packages
+        run: |
+          PACKAGES="$(git diff --diff-filter=d --name-only origin/master \
+            | grep 'Makefile$' | grep -Ev '/files/|/src/' \
+            | awk -F/ '{ print $(NF-1) }' | tr '\n' ' ')"
+
+          echo "Building $PACKAGES"
+
+          echo "::set-env name=PACKAGES::$PACKAGES"
+
+      - name: Build
+        uses: aparcar/action-openwrt-sdk@composite
```

Comment:
We should move that to OpenWrt repo / org

Reply:
What name would you prefer? `action-sdk` is okay?

Reply:
I'm bad at naming things, but maybe `gh-action-sdk` (action is a bit generic)

Reply:
Done